<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>1053</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HiveDesktop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Account info</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>Settings</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>account</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="accountLineEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QPushButton" name="refreshPushButton">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QCheckBox" name="autoRefreshCheckBox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Auto refresh</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="2">
            <widget class="QLabel" name="text2">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="text">
             <property name="font">
              <font>
               <pointsize>8</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lastUpvotesLabel">
          <property name="text">
           <string>Lastest Upvotes (last 24h)</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2">
         <widget class="QListWidget" name="lastUpvotesListWidget"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="curationRewardLabel">
          <property name="text">
           <string>Latest curation rewards (last 24h)</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QListWidget" name="lastCurationListWidget"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="authorRewardLabel">
          <property name="text">
           <string>Latest author rewards (last 24h)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QListWidget" name="lastAuthorListWidget"/>
        </item>
        <item row="0" column="1">
         <widget class="QGroupBox" name="accountInfoGroupBox">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="title">
           <string>username (70)</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="7" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="STEEMLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="SBDLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="SPLabel">
               <property name="font">
                <font>
                 <pointsize>10</pointsize>
                </font>
               </property>
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="votePowerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Vote Power</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" rowspan="2" colspan="2">
            <widget class="QProgressBar" name="downvotePowerProgressBar">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QProgressBar" name="RCProgressBar">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="value">
              <number>100</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignJustify|Qt::AlignVCenter</set>
             </property>
             <property name="format">
              <string>%p% full in 0 hours</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QProgressBar" name="votePowerProgressBar">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="value">
              <number>24</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="downvotePowerLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>DownVote Power</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="RCLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>RC</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Account history</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="1" column="0">
         <widget class="QListWidget" name="accountHistListWidget"/>
        </item>
        <item row="0" column="0">
         <widget class="QCheckBox" name="accountHistNotificationCheckBox">
          <property name="text">
           <string>Notification on all events</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Feed</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="2">
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QCheckBox" name="feedNotificationCheckBox">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Notification on all events</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QListWidget" name="feedListWidget"/>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="authorLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="titleLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="WebView" name="webview" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>794</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOptions"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAccount">
    <property name="title">
     <string>Account</string>
    </property>
   </widget>
   <addaction name="menuAccount"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionadd">
   <property name="text">
    <string>add</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WebView</class>
   <extends>QWidget</extends>
   <header>hivedesktop/widgets/webview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
